packages/core/getBusyTimes.ts

```diff
@@ -183,6 +196,17 @@ export async function getBusyTimes(params: {
       };
     });
 
+    if (rescheduleUid) {
```

Comment:
@Udit-takkar I could reproduce the issue you mentioned, however not always and it is working most of the time for me. I couldn't figure out yet why it sometimes isn't working but I think when it fails the rescheduleUid isn't coming in